images/spaceship.svg

```diff
@@ -1,15 +1,13 @@
-<svg width="200" height="200" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
-  <!-- Body of the spaceship -->
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" id="svgCanvas" version="1.1" height="128" width="128" viewBox="0 0 128 128" xml:space="preserve">
```

Comment:
Hello! For the spaceship
- restore dimensions: 200×200
- remove empty lines
- remove trailing spaces from line 12

Reply:
Thank you for your help! This is for a project where I need to teach kids that software developers use pipelines for validating!

I now have powershell version of your shell script that works on Windows:
https://github.com/danielsiegl/svgwebdemo/blob/10-improved-svg-validation-based-on-marketplace-action/utils/svg-format.ps1